lxd/daemon.go

```diff
@@ -545,90 +546,91 @@ func (d *Daemon) Init() error {
 	/* Detect user namespaces */
 	runningInUserns = shared.RunningInUserNS()
 
-	/* Detect AppArmor support */
-	if aaAvailable && os.Getenv("LXD_SECURITY_APPARMOR") == "false" {
-		aaAvailable = false
-		aaAdmin = false
+	/* Detect AppArmor availability */
+	_, err = exec.LookPath("apparmor_parser")
+	if os.Getenv("LXD_SECURITY_APPARMOR") == "false" {
```

Comment:
No and I don't really want to, it's only needed because of Snappy 15 and will die a horrible death as soon as we stop caring about that :)